src/venvstacks/stacks.py

```diff
@@ -1388,10 +1388,12 @@ def _create_new_environment(self, *, lock_only: bool = False) -> None:
     def _ensure_portability(self) -> None:
         # Wrapper and activation scripts are not used on deployment targets,
         # so drop them entirely rather than making them portable
-        for executable in self.executables_path.iterdir():
-            if not executable.name.lower().startswith("python"):
-                print(f"    Dropping potentially non-portable file {str(executable)!r}")
-                executable.unlink()
+        for item in self.executables_path.iterdir():
+            if item.is_dir():
+                shutil.rmtree(item)
```

Comment:
We should add logging here (once we fix the logging to actually be logging, instead of print statements)